styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -282,6 +282,8 @@ private void dequeueInstance0(final StyxConfig config, Map<String, Resource> res
       return;
     }
 
+    instanceResourceRefs.forEach(stats::recordResourceDemanded);
```

Comment:
This should perhaps use a histogram and emit aggregated counts instead. I believe a meter won't produce any intelligible numbers here. For example, if the scheduling interval changes, then the resource demand meter value would change, which doesn't make sense.

Reply:
On the other hand, does it make sense to interpret it as requested resources `now`?

Reply:
I guess what we want is numbers the gauge the "backlog" of instances that cannot run because they're blocking on an exhausted resource. Emitting an aggregate demand count would provide this.

https://github.com/spotify/styx/pull/650